squid/README.md

```diff
@@ -2,33 +2,21 @@
 
 ## Overview
 
-This integration lets you monitor your Squid metrics from the Cache Manager directly in Datadog.
+This check monitors [Squid][9] metrics from the Cache Manager through the Datadog Agent.
```

Comment:
Are we calling it a check or an integration?

Reply:
There are a lot of places we use `check` and `integration` as equivalent words. Not sure if we want to go with one or the other. The template for new Agent integrations uses the word `check`.